scss/atoms/modals/_modals.scss

```diff
@@ -0,0 +1,40 @@
+/*
+* ==========================================================================
+* Modals
+* ==========================================================================
+*/
+
+@import
+	"dist/chassis",
+	"functions",
+	"mixins";
+
+.modal {
+	@include modal();
+}
+
+.modal-header {
+	@include modal-header();
+}
+
+.modal-title {
+	margin: 0;
```

Comment:
You shouldn't need margin: 0 here because the mixin appears to also include a margin declaration.